src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -187,6 +189,10 @@ export class BeneficiariesService {
 						id: 'aadhaar_verification_mode',
 						title: 'Aadhaar Verification Mode',
 					},
+					{
+						id: 'aadhaar_verification_mode',
+						title: 'Aadhaar Verification Mode',
+					},
```

Comment:
Why this code block added? It is already there

Reply:
removed